docs/user/monitoring/monitoring-troubleshooting.asciidoc

```diff
@@ -56,9 +56,12 @@ from a different monitoring cluster, set `monitoring.ui.elasticsearch.hosts`.
 See <<monitoring-settings-kb>>.
 
 . Confirm that there is monitoring data available at that URL. It is stored in
-indices such as `.monitoring-kibana-*` and `.monitoring-es-*`. At a minimum, you
-must have monitoring data for the {es} production cluster. Once that data exists,
-{kib} can display monitoring data for other products in the cluster.
+indices such as `.monitoring-kibana-*` and `.monitoring-es-*` or
+`.ds-metrics-kibana.stack_monitoring-*` and
+`.ds-metrics-elasticsearch.stack_monitoring-*`, depending on which method is
```

Comment:
nit: we could refer to the data streams (ie `metrics-kibana.stack_monitoring-*`) instead of the backing indices (`.ds-..`)